src/PAModel/Serializers/MsAppSerializer.cs

```diff
@@ -227,7 +228,11 @@ public static CanvasDocument Load(Stream streamToMsapp, ErrorContainer errors)
                 // This is debug only. The server checksum is out of date with the client checksum
                 // The main checksum validation that matters is the repack after unpack
 #if DEBUG
```

Comment:
Now that we have an "is older" check, can we remove #if Debug and do this all the time?

Reply:
makes sense